packager/certloader/certloader.go

```diff
@@ -184,9 +174,14 @@ func WriteCertsToFile(certs []*x509.Certificate, filepath string) error {
 		}
 	}()
 
-	cert := certToPEM(certs[0])
-	issuer := certToPEM(certs[1])
-	bundled := append(cert, issuer...)
+	certLen := len(certs)
+	bundled := make([]byte, 0, 0)
+	for i := 0; i < certLen - 1; i++ {
+		cert := certToPEM(certs[i])
+		bundled = append(bundled, cert...)
+	}
+	issuer := certToPEM(certs[certLen])
```

Comment:
Won't `certs[certLen]` cause an out-of-range error?

Also, FYI, other than the leaf cert being at position 0, there's no ordering requirement for the rest of the certs array. The issuer cert can be any of the subsequent certs. (Hence why findIssuer() exists.)

Reply:
Ok fixed in previous comment.  Good to know about issuer position.